internal/resource/configmap.go

```diff
@@ -61,10 +61,12 @@ func (builder *ServerConfigMapBuilder) Update(object runtime.Object) error {
 	configMap := object.(*corev1.ConfigMap)
 	configMap.Labels = metadata.GetLabels(builder.Instance.Name, builder.Instance.Labels)
 	configMap.Annotations = metadata.ReconcileAndFilterAnnotations(configMap.GetAnnotations(), builder.Instance.Annotations)
+
 	if configMap.Data == nil {
 		configMap.Data = make(map[string]string)
 	}
-	configMap.Data["rabbitmq.conf"] = defaultRabbitmqConf
+
+	configMap.Data["rabbitmq.conf"] = defaultRabbitmqConf + fmt.Sprintln("cluster_name = "+builder.Instance.Name)
```

Comment:
Is the default config likely to expand in the future, in similar ways to this: requiring builder-specific data, and then appending config? If so, it may be worth moving the `const`s above to their own private method, e.g.

```go
func defaultRabbitmqConf(string instanceName) string { fmt.Sprintf(...) }
```

Reply:
If we refactor, might as well use a string [Builder](https://www.calhoun.io/concatenating-and-building-strings-in-go/) as well

Reply:
I'm not sure how likely is that we expand the default conf using builder data. In any case, I think it's good to have an option to do so. I will make this refactors 👌

Reply:
Implemented these changes in https://github.com/rabbitmq/cluster-operator/commit/a0025394f94d993443f7c8d865fdbb74902b7dd9 🚀